Assets/HoloToolkit/Input/Scripts/FocusedObjectMessageSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using HoloToolkit.Unity;
+
+public class FocusedObjectMessageSender : MonoBehaviour
```

Comment:
@NeerajW I think FocusedObjectMessageSender is great for the application level, but agree that it's overkill for the toolkit given it's simplicity.